activitysim/abm/models/school_escorting.py

```diff
@@ -369,6 +368,21 @@ class SchoolEscortSettings(BaseLogitComponentSettings, extra="forbid"):
         Multinomial logit model.
     """
 
+    @property
```

Comment:
Changing this to a property is the backwards fix: it keeps it readable but not writable.  But we need it to be writable (i.e. the values can appear in the YAML file and it will be allowed) but there should be a field validator that tells you about the deprecation.